chipsec/cfg/8086/pch_4xx.xml

```diff
@@ -31,6 +31,11 @@ chipsec@intel.com
   <info>
     <sku did="0xA3C8" name="B460" code="PCH_4xx" longname="400 series PCH B460" />
     <sku did="0xA3DA" name="H410" code="PCH_4xx" longname="400 series PCH H410" />
+    <sku did="0x0684" name="H470" code="PCH_4xx" longname="400 series PCH H470" />
+    <sku did="0x0685" name="Z490" code="PCH_4xx" longname="400 series PCH Z490" />
+    <sku did="0x0687" name="Q470" code="PCH_4xx" longname="400 series PCH Q470" />
+    <sku did="0x068C" name="QM480" code="PCH_4xx" longname="400 series PCH QM480" />
+    <sku did="0x0697" name="W480" code="PCH_4xx" longname="400 series PCH W480" />
   </info>

```

Comment:
These probably should be in **pch_4xxh.xml**

Reply:
Ah yes, indeed, sorry I didn't catch the `h` the first time. I moved the new config there in a new commit but I guess the commits can be squashed.